static/js/lib/api.js

```diff
@@ -272,3 +273,13 @@ export function patchProfileImage(
 export function getProfile(username: string): Promise<Profile> {
   return fetchJSONWithAuthFailure(`/api/v0/profiles/${username}/`)
 }
+
+export function updateProfile(
```

Comment:
Can you add a test for this function?

Reply:
Sure, test added. I also rebased with master & inserted `withSingleColumn` into the profile pages.